core/controllers/download.py

```diff
@@ -35,15 +35,18 @@ def download():
     rp_start_datetime = datetime.strptime(rp_start, DATETIME_ISO_8601) if rp_start else None
     rp_end_datetime = datetime.strptime(rp_end, DATETIME_ISO_8601) if rp_end else None
 
-    base_query = download_data_base_query(
+    query = download_data_base_query(
         rp_start_datetime,
         rp_end_datetime,
         organisation_ids,
         fund_ids,
         itl_regions,
-        outcome_categories,
     )
-    data_generator = serialise_download_data(base_query)
+
+    if outcome_categories:
+        query = query_extend_with_outcome_filter(query)
```

Comment:
This needs to be passed the outcome_categories param, this is why the outcomes filter is not working. Also, because the param defaults to None, we didn't notice it (when we test this stuff we did pass the param).